packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -233,37 +233,20 @@
 +##
 +ingressNginx:
 +  enabled: false
-+
```

Comment:
ingressNginx has never been exposed in a released chart (it is new to 2.5.8) so it's fine to completely change its YAML spec. There will be a dashboard change to update the necessary overrides.